### ISSUE TITLE
Add a cog that links to the settings page

### ISSUE THREAD

**Issue by samhotchkiss** [Automattic/jetpack#5688]
![image](https://cloud.githubusercontent.com/assets/574752/20370400/ba8216ac-ac1b-11e6-8ba1-294849644115.png)

Put it right on your Beau Lebens...

(Jeff, don't tell @rickybanister or @MichaelArestad -- just do it!)

**Comment by jeffgolenski** [comment]
Nay sir. Putting the settings link in here doesn't meet our design patterns. It's out of context there anyway. What we need to do is build out the proper top tier navigation like so...

<img width="866" alt="screen shot 2016-11-16 at 6 49 46 pm" src="https://cloud.githubusercontent.com/assets/214813/20370583/7d0d60e4-ac2d-11e6-807c-4eb50c83d32f.png">

Let's do this right. Squeezing little things in here and there without being methodical leads to a bad experience and more technical debt / code bloat. Let's talk about this for 4.5

**Comment by samhotchkiss** [comment]
Wherever you want as long as it's in the upper right corner...

**Comment by jeherve** [comment]
Closing as duplicate of #4821

**Comment by rickybanister** [comment]
I'm not convinced that top tier navigation is as visible as we think it is. That's not a pattern used in Calypso and not a place I'd look for a settings link. I can never even find screen options in wp-admin. 

I agree with @jeffgolenski that the search could be helpful. 

The similarity between the dashboard and settings pages can sometimes be confusing, add to that the search appearing and disappearing. Having search there, and perhaps the first 'search result' before anything is typed is a link to settings?

**Comment by jeffgolenski** [comment]
> I'm not convinced that top tier navigation is as visible as we think it is. That's not a pattern used in Calypso and not a place I'd look for a settings link. I can never even find screen options in wp-admin.

I think having that top tier nav is a heck of a lot more visible than having the settings link in the sidebar, where it currently resides, especially on tablets and smaller devices where the sidebar is hidden as a drawer nav. 

I'm against just haphazardly tossing things in places because there's an empty space. I definitely feel we should have the search icon here, just like we do in this area in the settings area, so the na structure is consistent. Folks should be able to quickly search for things while they're on the dashboard. If they use the search feature, it brings them to the settings anyway, and thus solved the issue of needing the cog icon there. (an argument might be that new users might not know that's how things work intitially, but after using it once it would become apparent)

I think we should have setting (cog icons) prompts in contextual spaces. Much like the security area lower in the dashboard. When they need to change settings for a feature they're looking at, there it is. But, perhaps like the older mockups I worked on, we can have them in less general areas. ie...

currently:
![screen shot 2016-11-17 at 4 20 30 pm](https://cloud.githubusercontent.com/assets/214813/20408040/cc295aca-ace1-11e6-8627-9a5ef68c42f5.png)

older mockup:
![screen shot 2016-11-17 at 4 27 09 pm](https://cloud.githubusercontent.com/assets/214813/20408269/c24c29fa-ace2-11e6-9f98-a20bdb1fc3a0.png)


@rickybanister I've dug around calypso a bit, but I'm unable to find any similar context we could utilize here. what are your thoughts on that?

Michael and I spent a bit of time working on the navigation issue in the past, but I think this is something we need to revisit and thing more on now that we're gathering data on usage.

cc @samhotchkiss @beaulebens
